lib/jekyll_pages_api/version.rb

```diff
@@ -1,3 +1,3 @@
 module JekyllPagesApi
-  VERSION = '0.1.1'
+  VERSION = '0.1.3'
```

Comment:
Pulled it down to 0.1.2. That version is released, but somehow hasn't yet been committed.